src/main/java/com/hotjava/app/hotjava/dao/PhotoDAO.java

```diff
@@ -10,30 +10,20 @@
 @Repository
 public class PhotoDAO implements IPhotoDAO {
 
-    //@Autowired
-   // private PhotoRepository photoRepository;
-
-   // @Autowired
-    //private KafkaTemplate<String, String> kafkaTemplate;
-
-
-
-
     @Override
     public void save(Photo photo) {
     //TODO photo repo
     }
 
     @Override
     public void saveImage(MultipartFile imageFile, Photo photo) throws IOException {
-       /* Path currentPath = Paths.get(".");
-        Path absolutePath = currentPath.toAbsolutePath();
-        photo.setPath(absolutePath + "/src/main/resources/static/photos/");
-        byte[] bytes = imageFile.getBytes();
-        Path path = Paths.get(photo.getPath() + imageFile.getOriginalFilename());
-        Files.write(path, bytes);
-        kafkaTemplate.send("photoIn", path.normalize().toString());*/
-        System.out.println("saveImage" + Integer.toString(photo.getPhotoId()));
```

Comment:
Moved to inside a try catch block where future exception handling will  occur. Functionality is not there yet, and this method does not do anything other than print text.